### DISCUSSION TITLE
Compiling HYDRUS HELP~~~

### DISCUSSION THREAD

**Discussion by leekari5548** [phydrus/phydrus#40] [Compiling Fortran Code]
Does anyone know which version of gfortran to compile into hydrus.exe or hydrus fil?
Mac or Windows version is ok.

**Comment by leekari5548** [comment]
ld: library not found for -lm
collect2: error: ld returned 1 exit status

**Comment by 1065844566** [comment]
Hi,
I wonder if you managed to solve this problem?
If so, can you send me a copy of the hydrus.exe you made? It needs to work for windows 10.
Please send it to my email address at 107556519106@stu.xju.edu.cn
Thanks！
